datalad/support/network.py

```diff
@@ -543,11 +553,34 @@ def _pr_to_fields(cls, pr):
             lgr.warning("ParseResults contains params %r, which will be ignored"
                         % (pr.params,))
 
+        hostname_port = pr.netloc.split('@')[-1]
```

Comment:
This one is just to split away possible login:pw@ prefix, leaving in only hostname and possibly port